api.ts

```diff
@@ -1037,10 +1039,19 @@ export interface SyncFormulaResult<
   L extends string,
   SchemaT extends ObjectSchemaDefinition<K, L>,
   ContextT extends SyncExecutionContext<any> = SyncExecutionContext,
+  PassthroughT extends SyncPassthroughData = SyncPassthroughData,
 > {
   /** The list of rows from this page. */
   result: Array<ObjectSchemaDefinitionType<K, L, SchemaT>>;
 
+  /**
+   * Some additional data that we don't  for each row in the result.
+   * This field is used to pass through data during the executeGetPermissions call
```

Comment:
Would probably do something like:

```
Additional side data to pass through to `executeGetPermissions`. Because each result item
is also passed through, this is only needed for cases where you have data that doesn't need
to be part of the result item but is nonetheless useful for fetching permissions, for example,
the API requests to get item data may include permissions inline, so there is no need to
re-fetch permission data if you can just pass it through.

This array must be the same length as the `result` array.
```